### PR TITLE
docs signature

### PR DESCRIPTION
**Pull request by adriacidre**
- [x] Creates new documents service to manage document signatures
- [x] Moves shared logic to `pkg/helpers`

### CODE REVIEW

File: chat/actions.go [hunk 1]
```diff
@@ -234,50 +233,6 @@ func (s *Service) send(recipients []string, req map[string]interface{}) error {
...
-				devices = append(devices, selfID+":"+dds[i])
-			}
-		}
-	}
-
-	return devices, nil
-}
-
 func (s *Service) createMissingSessions(members []string) error {
 	println("creating missing sessions 1")
```
Comment (purehyperbole) [inline chat/actions.go:237]: Not part of this PR, but I think you left some debug logging in before? :stuck_out_tongue:

File: documents/client.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+	"github.com/joinself/self-go-sdk/pkg/object"
+)
+
+// messagingClient handles all interactions with self messaging and its users
+type messagingClient interface {
+	Send(recipients []string, data []byte) error
+	Request(recipients []string, cid string, data []byte, timeout time.Duration) (string, []byte, error)
+	Subscribe(msgType string, sub func(sender string, payload []byte))
+	Command(command, selfID string, payload []byte) ([]byte, error)
```
Comment (purehyperbole) [inline documents/client.go:18]: Can any of these interface functions be removed? I can't see where `Command` and `ListConnections` are used for the document service

Comment (adriacidre) [reply documents/client.go]: yeah you're right, I've removed all but `Request` and `Command`

**Review by purehyperbole — APPROVED** [review]
